web/src/repo/TreePage.tsx

```diff
@@ -224,6 +227,33 @@ export class TreePage extends React.PureComponent<Props, State> {
     }
 
     public render(): JSX.Element | null {
+        const emptyElement = this.state.showOlderCommits ? (
+            <>No commits in this tree.</>
+        ) : (
+            <>
+                No commits in this tree in the past year.
+                <br />
+                <button type="button" className="btn btn-secondary btn-sm" onClick={this.showOlderCommits}>
+                    Show all commits
+                </button>
+            </>
+        )
+
+        const TotalCountSummary: React.FunctionComponent<{ totalCount: number }> = ({ totalCount }) => (
+            <div className="mt-2">
+                {this.state.showOlderCommits ? (
+                    <>{totalCount} total commits in this tree.</>
+                ) : (
+                    <>
+                        {totalCount} commit{totalCount > 1 ? 's' : ''} in this tree in the past year.
```

Comment:
```suggestion
                        {totalCount} {pluralize('commit', totalCount)} in this tree in the past year.
```

also handles the 0 case :)